.gitpod.yml

```diff
@@ -1,4 +1,22 @@
-# Commands to start on workspace startup. See https://www.gitpod.io/docs/config-gitpod-file
+# Commands to start on workspace startup. See https://www.gitpod.io/docs/config-start-tasks
 tasks:
   - init: yarn install && yarn run build
     command: yarn run watch
+
+github:
+  # https://www.gitpod.io/docs/prebuilds/#configure-prebuilds
+  prebuilds:
+    # enable for the default branch
+    master: true
+    # enable for all branches in this repo
+    branches: true
+    # enable for pull requests coming from this repo
+    pullRequests: true
+    # enable for pull requests coming from forks
+    pullRequestsFromForks: true
+    # add a check to pull requests
+    addCheck: true
+    # add a "Review in Gitpod" button to the pull request's description
+    addBadge: true
+    # don't add a "Review in Gitpod" button as a comment to pull requests
+    addComment: false
```

Comment:
Don't we need the github app for this to work?

Reply:
Yes, we need to install the gitpod GitHub app. could we install the GitHub app?

> To enable prebuilt workspaces for a GitHub repository, follow these steps:
> 
> Go to the [Gitpod GitHub app](https://github.com/apps/gitpod-io) and click Configure
> Choose the organization or account you wish to install the Gitpod app for, then click Install
> You will be forwarded to Gitpod where you can confirm the installation

see https://www.gitpod.io/docs/prebuilds#github

Reply:
I don't have the necessary privileges to do so

Reply:
anyone else we can contact to help?

Reply:
@JoelMarcey perhaps?

Reply:
@JoelMarcey could you help add the gitpod GitHub app to this repo?